### DISCUSSION TITLE
Trying to add custom block without reloading

### DISCUSSION THREAD

**Discussion by Rvdiaz012** [GrapesJS/grapesjs#5832] [Q&A] [answered]
I'm trying to add a custom block that was saved on my database but, I have to reload the page to be able to see it. Is there a way to show newly added blocks without reloading it?
here's the code, I'm using to test adding block by clicking a button.
`<script>
function addNewBlock() {
    editor.BlockManager.add('new-custom-block', {
    label: 'New Custom Block',
    content: newBlock,
    category: 'Custom Category'
    });
    editor.BlockManager.render();
}
</script>`

**Comment by artf** [comment, marked as answer]
You should be able to see it already, even without `BlockManager.render()`. Try to create a reproducible demo to confirm, in case it's not happening.

**Reply by Rvdiaz012** [reply]
Oh, I see, I'm just calling the wrong category name
https://codepen.io/bogartpoe/pen/xxemJWQ
